client/src/partials/fee_center/fee_center.js

```diff
@@ -0,0 +1,64 @@
+angular.module('bhima.controllers')
+.controller('FeeCenterController', FeeCenterController);
+
+FeeCenterController.$inject = ['FeeCenterService', 'NotifyService', '$state', 'SessionService'];
+
+function FeeCenterController(FeeCenterService, Notify, $state, Session) {
+  window.state = $state;
```

Comment:
We cannot expose internal values to the window - this allows it to be used by anyone.

I'm guessing this was for debugging - it should be removed.